sphinx/Infinitude_of_Primes.rst

```diff
@@ -5,26 +5,28 @@ This article gives a formal statement of Infinitude of Primes.  For Wikipedia's
 description, see
 `Infinitude of Primes <https://en.wikipedia.org/wiki/Prime_number#Infiniteness>`_.
 
-Informally statement
+Informally statement ::
 
   There are infinitely many :ref:`prime` numbers.
 
 
 Preformally ::
+  ..code-block::text
 
-  Prime number
+    Prime number
```

Comment:
The "preformalization" should just be a pseudocode version of the formal abstract, and in fact can be removed once you finish writing the formal abstract in Lean.

However, you have the right idea with spelling out the concepts you need to state this formal abstract. Maybe you could create definition pages for "prime" and "infinite set"?